components/Chat/Message/MessageReactionsList.tsx

```diff
@@ -153,7 +152,7 @@ const EmojiItem: FC<{
           alreadySelected && styles.selectedEmojiText,
         ]}
       >
-        <Text style={styles.emojiText}>{content}</Text>
+        <Text style={{ fontSize: 22, lineHeight: 28 }}>{content}</Text>
```

Comment:
Should we create a new text preset for this in Text component?

Reply:
@thierryskoda yes, great idea! love this preset feature. This is done!